pkgs/tools/compression/lzip/default.nix

```diff
@@ -27,6 +30,13 @@ stdenv.mkDerivation rec {
   doCheck = true;
   enableParallelBuilding = true;
 
+  installPhase =
+    if stdenv.hostPlatform.isMinGW then ''
+      sed -i '/^\s*$(INSTALL_PROGRAM) .\/$(progname) "\$(DESTDIR)$(bindir)\/$(progname)"$/s/$(progname)/$(progname).exe/g' Makefile
```

Comment:
I'd prefer this line to go to `postPatch`, and see if `installPhase` can be dropped completely and `outputs` made unconditional.

Reply:
Okay managed to get it the way you suggested. Way cleaner.